// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`.not.toHaveReceivedMessages > fails when called with an expected argument that is not a valid WS 1`] = `
[Error: [2mexpect([22m[31mWebSocketServer[39m[2m).not.toHaveReceivedMessages([22m[32mexpected[39m[2m)[22m

Expected the websocket object to be a valid WebSocketServer mock.
Received: string
  [31m"boom"[39m]
`;

exports[`.not.toHaveReceivedMessages > fails when the websocket server received at least one unexpected message 1`] = `
[Error: [2mexpect([22m[31mWebSocketServer[39m[2m).not.toHaveReceivedMessages([22m[32mexpected[39m[2m)[22m

Expected the WebSocketServer to not have received the following messages:

  [32mArray [
    "'sup?",
    "U good?",
    "hello there",
  ][39m

But it received:

  [31mArray [
    "hello there",
    "how are you?",
    "good?",
  ][39m
]
`;

exports[`.toHaveReceivedMessages > fails when called with an expected argument that is not a valid WS 1`] = `
[Error: [2mexpect([22m[31mWebSocketServer[39m[2m).toHaveReceivedMessages([22m[32mexpected[39m[2m)[22m

Expected the websocket object to be a valid WebSocketServer mock.
Received: string
  [31m"boom"[39m]
`;

exports[`.toHaveReceivedMessages > fails when the websocket server did not receive the expected messages 1`] = `
[Error: [2mexpect([22m[31mWebSocketServer[39m[2m).toHaveReceivedMessages([22m[32mexpected[39m[2m)[22m

Expected the WebSocketServer to have received the following messages:

  [32mArray [
    "hello there",
    "'sup?",
  ][39m

Received:

  [31mArray [
    "hello there",
    "how are you?",
    "good?",
  ][39m
]
`;
